api/proto/teleport/dbobjectimportrule/v1/dbobjectimportrule.proto

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.dbobjectimportrule.v1;
+
+import "teleport/header/v1/metadata.proto";
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/dbobjectimportrule/v1;dbobjectimportrulev1";
+
+// DatabaseObjectImportRule is the resource representing a global database object import rule.
+message DatabaseObjectImportRule {
+  // The kind of resource represented.
+  string kind = 1;
+  // Differentiates variations of the same kind. All resources should
+  // contain one, even if it is never populated.
+  string sub_kind = 2;
+  // The version of the resource being represented.
+  string version = 3;
+  // Common metadata that all resources share.
+  teleport.header.v1.Metadata metadata = 4;
+
+  // spec represents the specifications for the database object import rule.
+  DatabaseObjectImportRuleSpec spec = 5;
+}
+
+// DatabaseObjectImportRuleSpec is the spec for database object import rule.
+message DatabaseObjectImportRuleSpec {
+  // priority represents the priority of the rule application. Rules with higher priorities override those with lower.
```

Comment:
What is the scale for priorities? Is zero the highest or the bigger number the highest? (It seems like the latter but let's be 100% clear.)

Is zero a valid number?

Are negative numbers valid priorities?

Reply:
Yes, all integers are valid. How priorities are processed is explained in the context of the whole process, which is way too long to repeat in this place, but covered in RFD.

Reply:
Could we have a brief explanation here too? Pointing people to the RFD is fine, but it doesn't hurt saying whether negative is valid, or whether the highest priority is the highest number. We have more RFDs than most of us can keep up with, so let's try to make simple details avaible.